.github/workflows/security-scan.yml

```
@@ -0,0 +1,390 @@
+        if: matrix.component != 'backend'
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+          cache-dependency-path: |
+            frontend-nextjs/package-lock.json
+            e2e/package-lock.json
+
+      - name: Set up Python (for backend)
+        if: matrix.component == 'backend'
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)